CriticalMass/AppController.swift

```diff
@@ -60,7 +60,7 @@ class AppController {
     }
 
     private func getSocialViewController() -> SocialViewController {
-        return SocialViewController(chatViewController: getChatViewController, twitterViewController: getTwitterViewController)
+        return SocialViewController(chatViewController: getChatViewController(), twitterViewController: getTwitterViewController())
```

Comment:
What are the benefits here from removing the lazy loading?

Reply:
I don't see why they should be lazy. Actually I just changed the property inside the SocialController to be a `let` rarther then to be a closure so it makes code more readable in my eyes. What is the benefit here of having them lazy? It does not need to capture the AppController instance, right? Would opt-in for constants there anyhow I think

Reply:
Mostly smaller memory optimizations. By loading the content lazily only the current content is in memory and on switching the tab the old content get's released

Reply:
Hmm. That is true. Might also happen the user never opens the socialController. So that could be saved. I do think though that the pattern is mostly used for computational heavy allocations and if some properties are only there after initialization of the object that holds the lazy var. Did you ever profile the memory?